lib/tokenizers.js

```diff
@@ -110,8 +123,9 @@ export async function deprecateCurrent({explain = false} = {}) {
       query, {$set, $unset}, database.writeOptions);
     // return `true` if a tokenizer was marked as `deprecated`
     return result.result.n === 0;
-  } catch(e) {
-    throw e;
+  } catch(cause) {
+    throw new BedrockError('Error deprecating current tokenizer.',
+      'DatabaseError', null, cause);
```

Comment:
This also sounds like an invalid state error (also, `DatabaseError` isn't a choice from the MDN error list that we use).

```suggestion
      'InvalidStateError', null, cause);
```